Stripe/PublicHeaders/STPThreeDSCustomizationSettings.h

```diff
@@ -40,6 +40,8 @@ NS_ASSUME_NONNULL_BEGIN
  `authenticationTimeout` is the total time allowed for a user to complete a 3DS2 authentication
  interaction. This value *must* be at least 5 minutes.
  
+ @note The value is rounded down to the nearest minute.
```

Comment:
🤦‍♂ I retract my previous statement. You were correct to change this from an NSTimeInterval

Reply:
np!